WordPress/openverse
# Rename old data refresh to `legacy_data_refresh`

## Fixes



Related to #4146 by @stacimc 

## Description



This PR makes no changes to the current data refreshes, but renames the _directory_ to `legacy_data_refresh` to avoid naming collisions. I think this is the easiest path forward for simply deleting it when the new DAGs are running.

This is pulled into a separate PR because the diff for adding the new data refresh would otherwise be extremely confusing.

## Testing Instructions


Locally, run an image and an audio DAG for a few minutes to ingest some new records (I ran Cleveland and Jamendo). Now run the (regular) `audio_data_refresh` and `image_data_refresh` locally and verify that they still work. Note that the reported record count will have increased because you got some new local data before running them.


## Checklist


      PRs) or the media properties generator (`just catalog/generate-docs media-props`
      for the catalog or `just api/generate-docs` for the API) where applicable.

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
